website/docs/guides/core-to-cloud-1.md

```diff
@@ -119,42 +131,46 @@ This section highlights the development configurations you’ll need for your db
+- Making it easy to configure environments through the dbt Cloud UI instead of manually editing the `profiles.yml` file. You can also [set up](/reference/dbt-jinja-functions/target) or [customize](/docs/build/custom-target-names) target names in dbt Cloud.
+- Adding `profiles.yml` attributes to dbt Cloud environment settings with [Extended Attributes](/docs/dbt-cloud-environments#extended-attributes).
+- Using [Git repo caching](/docs/dbt-cloud-environments#git-repository-caching) to protect you from third-party outages, Git auth failures, and more. <Lifecycle status="enterprise"/>
 
 ### Initial setup steps
 1. **Set up development environment** &mdash; Set up your [development](/docs/dbt-cloud-environments#create-a-development-environment) environment and [development credentials](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud#access-the-cloud-ide). You’ll need this to access your dbt project and start developing.
 
 2. **dbt Core version** &mdash; In your dbt Cloud environment and credentials, use the same dbt Core version you use locally. You can run `dbt --version` in the command line to find out which version of dbt Core you’re using.
-   - Once your full migration is complete, consider upgrading your environments to [Keep on latest version](/docs/dbt-versions/upgrade-dbt-version-in-cloud#keep-on-latest-version-) to always get the latest features and more.
+   - When using dbt Core, you need to think about which version you’re using and manage your own upgrades. When using dbt Cloud, leverage [Keep on latest version](/docs/dbt-versions/upgrade-dbt-version-in-cloud#keep-on-latest-version-) so you don’t have to.
 
 3. **Connect to your data platform** &mdash; When using dbt Cloud, you can [connect to your data platform](/docs/cloud/connect-data-platform/about-connections) directly in the UI.
    - Each environment is roughly equivalent to an entry in your `profiles.yml` file. This means you don't need a `profiles.yml` file in your project.
 4. **Development tools** &mdash; Set up your development workspace with the [dbt Cloud CLI](/docs/cloud/cloud-cli-installation) or [dbt Cloud IDE](/docs/cloud/dbt-cloud-ide/develop-in-the-cloud) to edit and develop your dbt code in your tool of choice.
 
 ### Additional configuration
 Explore these additional configurations to optimize your developer setup further:
-1. **Custom target names** &mdash; If you’re using a [`custom target.name`](/reference/dbt-jinja-functions/target) in your project, we recommend you set them using [environment variables](/docs/build/environment-variables). Alternatively, you can update it at the developer credentials level.
+1. **Custom target names** &mdash; If you’re using a [`custom target.name`](/reference/dbt-jinja-functions/target) in your project, we recommend setting them at the developer credentials level. Alternatively, you can set them using [environment variables](/docs/build/environment-variables).
```

Comment:
Should we mention why we recommend one method over the other or do the linked docs cover that?

Reply:
i think it depends on the user and their set up. but this was the recommended order by @jtcohen6 in terms of what is best practice for developers and what isn't.

Reply:
@mirnawong1 I had a follow-up conversation ([internal slack](https://dbt-labs.slack.com/archives/C06RHGZ96DT/p1712334965538109?thread_ts=1712251385.984029&cid=C06RHGZ96DT)) with @graciegoheen, who (along with @b-per) is team "always env var never `target.name`." I think I can get on board :)

We aligned on the following path forward:
1. Encourage users to **always** use env vars (preferred), rather than defining env-conditional logic based on `target.name` (legacy)
2. Update the default implementation of [`generate_schema_name_for_env`](https://github.com/dbt-labs/dbt-adapters/blob/7392debf7ff1aa5d58c0fe26eb76ac8ade4fe15b/dbt/include/global_project/macros/get_custom_name/get_custom_schema.sql#L50) to check a specific env var (if set) in addition to `target.name == 'prod'`
3. In dbt Cloud, set that env var on users' behalf (`"dev|ci|staging|prod"`)

IMO - we should follow up here once we've done (2) + (3). Until then, we cannot wholeheartedly endorse (1), but we can still suggest that env vars are preferred over `{{ target }}` logic wherever possible, because:
- cleaner separation of concerns (env vars are for managing environments, `target` is really for defining the DWH connection)
- [partial parsing](https://docs.getdbt.com/reference/parsing#known-limitations) intelligently supports changes in env vars, and doesn't have the same support for `target` values

Reply:
@mirnawong1 Good for now, and let's take a note (somewhere) to follow up in a few months when we have a clearer story here.

Reply:
great, thanks Jeremy! added an issue here: https://github.com/dbt-labs/docs.getdbt.com/issues/5252